examples/react/src/App.js

```diff
@@ -0,0 +1,148 @@
+import React, { Component } from 'react';
+import {Router} from 'director';
+
+import TodoFooter from './Footer';
+import TodoItem from './TodoItem';
+import {ALL_TODOS, ACTIVE_TODOS, COMPLETED_TODOS} from './utils'
+
+const ENTER_KEY = 13;
+
+class App extends Component {
+  state = {
+      nowShowing: ALL_TODOS,
```

Comment:
Please format the project using [Prettier](https://prettier.github.io/prettier/).
In particular, we can see some mismatching indentation here.